src/sestest/VerifyCommand.cs

```diff
@@ -43,24 +43,21 @@ public VerifyCommand(ILogger logger) : base(logger)
             Errorable<string> app = FindApplication(commandLine);
             Errorable<string> api = FindApiVersion(commandLine);
 
-            Errorable<string> result = await server.With(token)
+            var result = await server.With(token)
```

Comment:
I'd suggest switching this back to `Errorable<string>` - knowing that the result is an `Errorable<T>` is helpful to someone reading the code.

Reply:
Oh, that should probably be `Errorable<int>`, given the other changes.